Website/htdocs/mpmanager/resources/assets/js/components/MiniGrid/Dashboard.vue

```diff
@@ -759,9 +758,25 @@
                         }
                         tmpChartData.push(totalRev)
                         this.trendChartData.base.push(tmpChartData)
+
                     }
+                    for (let i = 0; i < this.trendChartData.base.length; i++) {
 
+                        let trendChartData = this.trendChartData.base[i]
+                        let columnNumber = this.trendChartData.base[0].length
+                        if (i !== 0) {
+                            if (trendChartData.length > columnNumber) {
 
+                                trendChartData.splice(trendChartData.length - 1, trendChartData.length - columnNumber)
```

Comment:
Kicking out without knowing what is been deleted from the array is not an ideal solution.
Please revert that change.